mgmtd/mgmt_txn.c

```diff
@@ -2138,6 +2138,7 @@ int mgmt_txn_notify_be_adapter_conn(struct mgmt_be_client_adapter *adapter,
 		if (!txn) {
 			MGMTD_TXN_ERR("Failed to create CONFIG Transaction for downloading CONFIGs for client '%s'",
 				      adapter->name);
+			nb_config_diff_del_changes(adapter_cfgs);
```

Comment:
shouldn't this be it's own PR?

Reply:
Oh we're working constantly with Chris to fix problems in mgmtd that weren't noticed before because it had only a single backend daemon. So we include fixes into daemon-specific PR to test that everything works in CI. We can probably open a separate PR only with the fixes after with catch them all.